pkg/auth/auth.go

```diff
@@ -16,9 +16,9 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
-// GetDefaultAuthFile returns env value REGISTRY_AUTH_FILE as default --authfile path
+// getDefaultAuthFile returns env value REGISTRY_AUTH_FILE as default --authfile path
 // used in multiple --authfile flag definitions
-func GetDefaultAuthFile() string {
+func getDefaultAuthFile() string {
```

Comment:
We could use this for other APIs
```
podman create --help
...
       --authfile=path

       Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json

       Note: You can also override  the  default  path  of  the  authentication  file  by  setting  the  REG‐
       ISTRY_AUTH_FILE  environment  variable.  export REGISTRY_AUTH_FILE=path (Not available for remote com‐
       mands)

```

Reply:
It would be good to set the default location so users could see it.

Reply:
So should I drop these parts? The approval confuses me :)

Reply:
Yes drop this patch and we can merge.

Reply:
Dropped both making both this and `CheckAuthFile` private.